src/main/java/bio/terra/common/ValidateBucketAccessStep.java

```diff
@@ -36,7 +37,12 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
       sourcePath = List.of(((BulkLoadRequestModel) loadModel).getLoadControlFile());
     } else if (loadModel instanceof IngestRequestModel) {
       // Metadata ingests
-      sourcePath = List.of(((IngestRequestModel) loadModel).getPath());
+      // Don't validate if we are ingesting as a payload object
+      if (((IngestRequestModel) loadModel).getFormat().equals(FormatEnum.ARRAY)) {
```

Comment:
IngestUtils.isIngestFromPayload()?

Reply:
I can't sadly since it's using a flight context...I was debating abstracting more but thought that might get to be a lot for a single enum check